docs/api-reference/api-reference.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: api-reference
-title: HTTP API endpoints reference
+title: HTTP API reference
 sidebar_label: API endpoints reference
```

Comment:
```suggestion
sidebar_label: Overview
```
Since we need to keep the file name `api-reference.md`, suggest renaming the sidebar to "Overview" in line with other sections. That way API reference is not repeated in the sidebar. In any case, it's not just endpoints.
<img width="257" alt="Screenshot 2023-06-08 at 10 24 12 AM" src="https://github.com/apache/druid/assets/38529548/0735b9f9-e495-4dbb-aa19-b99982b970b3">